### PR TITLE
Fix importing in an addon and transfer loading-scroll-view

### PR DESCRIPTION
**Pull request by mattmcmanus**

### CODE REVIEW

File: addon/components/loading-scroll-view.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import Ember from 'ember';
+import ScrollView from './scroll-view';
+const { observer } = Ember;
+
+const THRESHOLD_FROM_BOTTOM_TO_TRIGGER_LOAD_MORE = 350;
+const MAX_LOAD_MORE_FREQUENCY_MS = 1000;
+
+export default ScrollView.extend({
+  scrollableRegistry: null, //injected
```
Comment (lukemelia) [inline addon/components/loading-scroll-view.js]: Let's just `inject` a la https://github.com/yappbox/yapp-scroll-view/blob/8221d1d945b52f098aa3eab806fe88305df87f03/addon/mixins/scroller-measurement.js#L12

Comment (mattmcmanus) [inline addon/components/loading-scroll-view.js]: Since it's already injected with the mixin, I reckon we can just remove the line outright, no?

Comment (lukemelia) [inline addon/components/loading-scroll-view.js]: 👍

Comment (mattmcmanus) [inline addon/components/loading-scroll-view.js]: Ok, updated and working

**Pull request merged**